src/ir/auth_logic/ast.h

```diff
@@ -400,9 +400,9 @@ class Program {
       query_strings.push_back(query.DebugPrint());
     }
     return absl::StrCat("Program(\n",
-                        absl::StrJoin(relation_decl_strings, "\n"),
-                        absl::StrJoin(says_assertion_strings, "\n"),
-                        absl::StrJoin(query_strings, "\n"), ")");
+                        absl::StrJoin(relation_decl_strings, "\n"), "\n",
+                        absl::StrJoin(says_assertion_strings, "\n"), "\n",
+                        absl::StrJoin(query_strings, "\n"), "\n)");
```

Comment:
Rather than `StrCat`, you might want to just join these things with another `StrJoin` with a delimiter of `\n`.

Reply:
fixed